xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/main/java/org/xwiki/rendering/internal/macro/DefaultMacroManager.java

```diff
@@ -99,7 +99,7 @@
             } catch (ParseException e) {
                 // One of the macros is registered against the component manager with an invalid macro id, ignore it
                 // but log a warning.
-                this.logger.warn("Invalid Macro descriptor format for hint [ {}" , entry.getKey()
+                this.logger.warn("Invalid Macro descriptor format for hint [ {}" , entry.getKey(), 
```

Comment:
Again you did not understood. The whole message is supposed to be the first parameter and then you have entry.getKey() as second parameter (which will replace the `{}`).

Reply:
Ohh